### PR TITLE
weekly ICR json data generation

### PR DESCRIPTION
**Pull request by korpx-z**
New images added (including httpd & HAProxy)
version updated where applicable as well.

(generated weekly by jenkins)

Signed-off-by: Charles Korpics [charleskorpics@gmail.com](mailto:charleskorpics@gmail.com)

### CODE REVIEW

File: data_files/IBMZ-container-registry.json [hunk 1]
```diff
@@ -0,0 +1,672 @@
...
+  },
+  {
+    "packageName": "alpine",
+    "version": "3.6",
+    "description": "Alpine Linux is a Linux distribution."
+  },
+  {
+    "packageName": "alpine",
+    "version": "3.7",
+    "description": "Alpine Linux is a Linux distribution."
```
Comment (rachejazz) [inline data_files/IBMZ-container-registry.json]: Is the file intended to have multiple versions of one package?Shouldn't it be the latest version per package?

File: data_files/IBMZ-container-registry.json [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+  },
+  {
+    "packageName": "debian",
+    "version": "stable",
+    "description": "Debian is an operating system which is composed primarily of free and open-source software, most of which is under the GNU General Public License, and developed by a group of individuals known as the Debian project."
+  },
+  {
+    "packageName": "debian",
+    "version": "stable-slim",
+    "description": "Debian is an operating system which is composed primarily of free and open-source software, most of which is under the GNU General Public License, and developed by a group of individuals known as the Debian project."
```
Comment (rachejazz) [inline data_files/IBMZ-container-registry.json]: Also, should debian be part of the package list? It has several versions too

**Comment by rachejazz** [comment]
This PR also has several conflicts, with versions mostly

File: data_files/IBMZ-container-registry.json (hunk 1 above)
Comment (pleia2) [reply data_files/IBMZ-container-registry.json]: Yes, we decided that was the best way for it to show that there are multiple versions available in the registry, so users have a choice of which to install.

File: data_files/IBMZ-container-registry.json (hunk 2 above)
Comment (pleia2) [reply data_files/IBMZ-container-registry.json]: These are container images, these ones are just basic Debian images :+1:

**Comment by pleia2** [comment]
@korpx-z Can you have a look at the conflicts?

**Comment by korpx-z** [comment]
Sorry about that, I meant to do this yesterday! @rachejazz thanks for resolving some of these btw

**Review by pleia2 — APPROVED** [review]

**Pull request merged**